Charts/Classes/Data/Implementations/Standard/ChartDataEntry.swift

```diff
@@ -38,30 +38,32 @@ public class ChartDataEntry: NSObject
         self.xIndex = xIndex
     }
     
-    public init(value: Double, xIndex: Int, data: AnyObject?)
+    public init(value: Double, xIndex: Int, data: Any?)
```

Comment:
I think it is. Its proposal [SE-0116](https://github.com/apple/swift-evolution/blob/master/proposals/0116-id-as-any.md)